apps/main/src/dex/components/PageRouterSwap/components/DetailInfoTradeRoute.tsx

```diff
@@ -33,7 +44,7 @@ const DetailInfoTradeRoute = ({
         padding={isMultiRoutes ? '0 0 0 var(--spacing-1)' : ''}
       >
         {loading ? (
-          <Loader skeleton={[130, 22.6 * (routesLength || 1)]} />
+          <Loader skeleton={[180, 22.6 * (routesLength || 1)]} />
```

Comment:
I made the routes component a little larger in the router page. Also in the non-beta version. It wouldn't fit the existing information.